test/base.test.js

```diff
@@ -346,32 +355,26 @@ describe('Base', function() {
             });
 
             it('retries 429s until success', function() {
-                const realSetTimeout = setTimeout;
-
-                jest.useFakeTimers();
```

Comment:
why was this changed? & number of requests changed from 3 to 2

Reply:
It was a few weeks ago, so I'm not 100% sure. But I think I just changed it from 2 to 3 requests to make the tests faster.

As work that procceded, I wrote a bunch of new timeout tests (to get 100% coverage) that use the actual timeouts and I think I just made this test more consistent with the other. 

I think I did this work because I ran into an issue with how I passed the timeout signal with the new node fetch library which caused me to dig in with these tests more.

Reply:
Ahh i see - sounds reasonable, thanks!